en.lproj/Localizable.strings

```diff
@@ -0,0 +1,13 @@
+/* 
+  Localizable.strings
+  TaskManager
+
+  Created by user on 28.09.2022.
+  
+*/
+
+"TaskLabel" = "Task Manager";
```

Comment:
the structure should like this 
"home.activeSection" = "Active";
"home.editButton" = "Edit";
usually with this files work People that translate the app and for them its easier to understand with such structure where the string is used

Reply:
this isnt updated